src/Webserver/BuiltInWebserverController.php

```diff
@@ -46,10 +46,11 @@ private function getCommand()
     {
         $phpFinder = new PhpExecutableFinder();
         return sprintf(
-             'exec %s -S %s -t %s',
+             'exec %s -S %s -t %s %s',
              escapeshellcmd($phpFinder->find()),
              escapeshellarg($this->config->getHost() . ':' . $this->config->getPort()),
-             escapeshellarg($this->config->getDocroot())
+             escapeshellarg($this->config->getDocroot()),
+             escapeshellarg($this->config->getDocroot() . '/' . $this->config->getRouter())
```

Comment:
Yes I agree, I didn't realise that but I saw you other comment too and you are right.